docs/source/demos/lead_scoring.ipynb

```diff
@@ -133,7 +133,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Because the lead scoring labels are binary, we will use `AutoMLSearch(X_train=X_train, y_train=y_train, problem_type='binary')`. When we call `.search()`, the search for the best pipeline will begin. "
+    "Because the lead scoring labels are binary, we will use set the problem type to \"binary\". When we call `.search()`, the search for the best pipeline will begin. "
```

Comment:
It was bugging me that `AutoMLSearch(X_train=X_train, y_train=y_train, problem_type='binary')` is not how we actually create automl in the next cell.